src/devices_models/devices/hydro_generation.jl

```diff
@@ -614,6 +639,7 @@ function add_constraints!(
 ) where {V <: PSY.HydroGen, W <: AbstractHydroFormulation, X <: PM.AbstractPowerModel}
     time_steps = get_time_steps(container)
     resolution = get_resolution(container)
+    # TODO: Check why this isn't used
```

Comment:
@sourabhdalvi why aren't we using this variable for unit conversions?

Reply:
This is in the EnergyTarget constraint, so we are comparing Energy with Energy, so no need to for unit conversion.